src/js/components/MaskedInput/StyledMaskedInput.js

```diff
@@ -0,0 +1,51 @@
+import styled, { css } from 'styled-components';
+
+import { focusStyle, inputStyle } from '../../utils';
+
+const placeholderColor = css`
+  color: ${props => props.theme.global.colors.placeholder};
+`;
+
+const sizeStyle = props => {
+  const data = props.theme.text[props.size];
+  return css`
+    font-size: ${data.size};
+    line-height: ${data.height};
+  `;
+};
+
+const plainStyle = css`
+  border: none;
+`;
+
+export const StyledMaskedInput = styled.input`
+  ${inputStyle} width: 100%;
+
+  ${props => props.size && sizeStyle(props)} ${props =>
+    props.plain && plainStyle}
+
+  &::-webkit-input-placeholder {
```

Comment:
I've seen this code elsewhere, should we create a `placeholderStyle` and reuse it ?

Reply:
Done.